data_model/master/clusters/ServiceArea.xml

```diff
@@ -1,59 +1,59 @@
-sublicense), under Connectivity Standards Alliance's applicable copyright
-rights, to view, download, save, reproduce and use the document solely for your
-own internal purposes and in accordance with the terms of the license set forth
-herein. This license does not authorize you to, and you expressly warrant that
-you shall not: (a) permit others (outside your organization) to use this
-document; (b) post or publish this document; (c) modify, adapt, translate, or
-otherwise change this document in any manner or create any derivative work
-based on this document; (d) remove or modify any notice or label on this
-document, including this Copyright Notice, License and Disclaimer. The
-Connectivity Standards Alliance does not grant you any license hereunder other
-than as expressly stated herein.
-
-Elements of this document may be subject to third party intellectual property
-rights, including without limitation, patent, copyright or trademark rights,
-and any such third party may or may not be a member of the Connectivity
-Standards Alliance. Connectivity Standards Alliance members grant other
-Connectivity Standards Alliance members certain intellectual property rights as
-set forth in the Connectivity Standards Alliance IPR Policy. Connectivity
-Standards Alliance members do not grant you any rights under this license. The
-Connectivity Standards Alliance is not responsible for, and shall not be held
-responsible in any manner for, identifying or failing to identify any or all
-such third party intellectual property rights. Please visit www.csa-iot.org for
-more information on how to become a member of the Connectivity Standards
-Alliance.
-
-This document and the information contained herein are provided on an “AS IS”
-basis and the Connectivity Standards Alliance DISCLAIMS ALL WARRANTIES EXPRESS
-OR IMPLIED, INCLUDING BUT NOT LIMITED TO (A) ANY WARRANTY THAT THE USE OF THE
-INFORMATION HEREIN WILL NOT INFRINGE ANY RIGHTS OF THIRD PARTIES (INCLUDING
-WITHOUT LIMITATION ANY INTELLECTUAL PROPERTY RIGHTS INCLUDING PATENT, COPYRIGHT
-OR TRADEMARK RIGHTS); OR (B) ANY IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS
-FOR A PARTICULAR PURPOSE, TITLE OR NONINFRINGEMENT. IN NO EVENT WILL THE
-CONNECTIVITY STANDARDS ALLIANCE BE LIABLE FOR ANY LOSS OF PROFITS, LOSS OF
-BUSINESS, LOSS OF USE OF DATA, INTERRUPTION OF BUSINESS, OR FOR ANY OTHER
-DIRECT, INDIRECT, SPECIAL OR EXEMPLARY, INCIDENTAL, PUNITIVE OR CONSEQUENTIAL
-DAMAGES OF ANY KIND, IN CONTRACT OR IN TORT, IN CONNECTION WITH THIS DOCUMENT
-OR THE INFORMATION CONTAINED HEREIN, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
-LOSS OR DAMAGE.
-
-All company, brand and product names in this document may be trademarks that
-are the sole property of their respective owners.
-
-This notice and disclaimer must be included on all copies of this document.
-
-Connectivity Standards Alliance
-508 Second Street, Suite 206
-Davis, CA 95616, USA
+<!--
```

Comment:
Please revert changes to this file - these are automatically generated from the spec and will be updated before SVE.

Reply:
I have no idea why this is happening. This is an exact copy of what's currently on master.

Reply:
would it be text encoding shenanigans? But I guess you can still revert/checkout this file from master an push it so github doesn't diff this